onadata/libs/utils/export_builder.py

```diff
@@ -399,8 +400,10 @@ def get_choice_dict(xpath, label):
             }
 
         choices = []
-
-        if not child.children and child.choice_filter and child.itemset:
+        is_choice_randomized = str_to_bool(
+            child.parameters and child.parameters.get('randomize'))
+        if ((not child.children and child.choice_filter)
+                or is_choice_randomized) and child.itemset:
```

Comment:
I think we are missing a test for this change, we could test `_get_select_mulitples_choices` in the `test_export_builder`.